### PR TITLE
66 set up instructor and practitioner view page

### PR DESCRIPTION
**Pull request by qstommyshu**

### CODE REVIEW

**Review by jiaoa3 — APPROVED** [review]
approving this pr

File: src/client/src/modules/RTCControl.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// TODO: Need to update after we have consumer side annotation selection to server
+// and need to update with data channel after we have dynamic annotation selection
+export const connectAsConsumer = async (pc: RTCPeerConnection) => {
+  const offer = await pc?.createOffer();
+  await pc?.setLocalDescription(offer);
+  const requestSdp = pc.localDescription;
+  const sdp = await fetch('http://127.0.0.1:8080/consumer', {
+    body: JSON.stringify({
+      sdp: requestSdp?.sdp,
+      type: requestSdp?.type,
```
Comment (yex33) [inline src/client/src/modules/RTCControl.ts:19]: Missing "video-transform" param here.

File: src/client/src/modules/RTCControl.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    sdpSemantics: 'unified-plan',
+    iceServers: [{ urls: 'stun:stun.l.google.com:19302' }],
+  };
+  const pc = new RTCPeerConnection(config);
+  return pc;
+};
+
+// TODO: Need to update after we have consumer side annotation selection to server
+// and need to update with data channel after we have dynamic annotation selection
+export const connectAsConsumer = async (pc: RTCPeerConnection) => {
```
[the remark this answers is not in the document]
Comment (qstommyshu) [reply src/client/src/modules/RTCControl.ts]: fixed

File: src/client/src/modules/RTCControl.ts (hunk 1 above)
Comment (qstommyshu) [reply src/client/src/modules/RTCControl.ts:17]: resolved

File: src/client/src/modules/mediaControl.tsx
Comment (yex33) [inline src/client/src/modules/mediaControl.tsx:1]: remove this empty file

Comment (qstommyshu) [reply src/client/src/modules/mediaControl.tsx:1]: resolved

**Review by yex33 — APPROVED** [review]

**Pull request merged**